week-4/InClass/F-object-keys/exercise-part-1.js

```diff
@@ -15,9 +15,8 @@ let mentorsAges = {
 
 // ONLY EDIT BELOW THIS LINE
 
-let mentorsNames = ;
-
-let mentorsNamedUppercased = ;
+let mentorsNames = Object.keys(mentorsAges);
+let mentorsNamedUppercased = mentorsNames.toUpperCase();
```

Comment:
This one doesn't quite work like this - `mentorNames` is an `Array`, and `toUpperCase` is a method on things with type `string` - can you think how to rewrite this so that you're applying `toUpperCase` to each element _inside_ `mentorNames`, rather than trying to apply it to the array itself?

You've done this before, so hopefully you can work it out again! :)